homeassistant/helpers/frame.py

```diff
@@ -400,6 +395,25 @@ def _report_integration_frame(
     )
 
 
+def _report_integration_unknown(
+    what: str,
+    core_behavior: ReportBehavior,
+    breaks_in_ha_version: str | None,
+    err: MissingIntegrationFrame | None,
+) -> None:
+    """Report incorrect usage when an integration is not identified."""
```

Comment:
The docstring is confusing. It sounds as if we're reporting because an integration wasn't found. But that's not what we mean I think.